test/built-ins/Temporal/Instant/prototype/since/options-may-be-function-object.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant.prototype.since
+description: Options may be a function object.
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const feb20 = Temporal.Instant.from("2020-02-01T00:00Z");
+const feb21 = Temporal.Instant.from("2021-02-01T00:00Z");
+
+TemporalHelpers.assertDurationsEqual(feb21.since(feb20, () => {}),
+                                     new Temporal.Duration(0, 0, 0, 0, 0, 0, 31622400, 0, 0, 0));
```

Comment:
```suggestion
TemporalHelpers.assertDuration(feb21.since(feb20, () => {}),
                                     0, 0, 0, 0, 0, 0, 31622400, 0, 0, 0);
```
(same suggestion for `until/`)